README.md

```diff
@@ -1,4 +1,6 @@
 # libevent-rs
+[![](https://meritbadge.herokuapp.com/libevent)](https://crates.io/crates/libevent)
```

Comment:
I've never seen meritbadge, I've always used shields.io. Are there any pros/cons to one vs the other?

Reply:
I just copy-pasta'ed from the `libevent-sys` README for ~laziness~ consistency. And looking at my other crates, some use shields.io while others use meritbadge, so there's really no distinct reasoning behind the choice. Probably good just to pick one and stick with it though 🤷